src/assets/styles/AddPet.css

```diff
@@ -0,0 +1,98 @@
+h2 {
```

Comment:
be careful of use these tags. I recommend to add parent class and after that add h2 tag

Reply:
Thank you Leonel I update the classnames and the css so they don't create conflicts with other sections of the project.